crates/runtime/src/flight/flightsql/prepared_statement_query.rs

```diff
@@ -86,8 +110,6 @@ pub(crate) async fn get_flight_info(
 
     let info = FlightInfo::new()
         .with_endpoint(endpoint)
-        .try_with_schema(&arrow_schema)
-        .map_err(to_tonic_err)?
         .with_descriptor(fd);
```

Comment:
Why was this removed?

Reply:
I noticed that the schema at this point says the field is nullable. The schema returned in the `DoGet` has the field as not nullable. I tested the same client code against DuckDB on the particular query and it returned not nullable. So I concluded that this schema here is incorrect and found that removing it fixed the issue. Do you think we need to fix the schema calculation here and leave this part in?

```shell
adbc_driver_manager.OperationalError: UNKNOWN: [FlightSQL] endpoint 0 returned inconsistent schema: expected schema:
  fields: 1
    - the_answer: type=int64, nullable but got schema:
  fields: 1
    - the_answer: type=int64
 ```

Reply:
We should understand why the schema is different. Also if we're not going to return the schema here, then we don't need to calculate it above.